src/Sarif.Driver/Sdk/ExportRulesDocumentationCommandBase.cs

```diff
@@ -46,13 +48,20 @@ internal void BuildRule(Skimmer<TContext> rule, StringBuilder sb)
         {
             sb.AppendLine($"## Rule `{rule.Moniker}`{Environment.NewLine}");
             sb.AppendLine($"### Description{Environment.NewLine}");
-            sb.AppendLine($"{rule.FullDescription.Text}{Environment.NewLine}");
+            sb.AppendLine($"{rule.FullDescription.Text ?? rule.ShortDescription.Text ?? "No description available."}{Environment.NewLine}");
```

Comment:
>Text [](start = 50, length = 4)

Prefer Markdown description to text if present. So IMO it should be
```
rule.FullDescription.Markdown ?? rule.FullDescription.Text ?? rule.ShortDescription.Markdown ?? rule.ShortDescription.Text
```

Reply:
And need test cases for this.

---
In reply to: [493035978](https://github.com/microsoft/sarif-sdk/pull/2082#discussion_r493035978) [](ancestors = 493035978)